.github/workflows/psammead-change-scanner.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           persist-credentials: false
 
       - name: Install Node Modules
-        run: yarn ci
+        run: ./scripts/yarnInstallCi.sh
```

Comment:
this is my opinion but I liked explicitly seeing these steps. keen to hear what other think though

Reply:
See https://github.com/bbc/simorgh/pull/9318#discussion_r690294866